### PR TITLE
Ay/tempo

### PR DESCRIPTION
**Pull request by ahcyip**

### CODE REVIEW

File: project/dimensions/enduses.csv [hunk 1]
```diff
@@ -16,10 +16,17 @@ electricity_refrigeration,Refrigeration,electricity,MWh
...
-electricity_ev_commercial_dcfc,EV Commercial DCFC,electricity,MWh
-electricity_ev_commercial_l1l2,EV Commercial L1/L2,electricity,MWh
-electricity_ev_residential_l1l2,EV Residential L1/L2,electricity,MWh
-electricity_lightrail,Light Rail,electricity,MWh
+electricity_ev_res1,LD EV Residential L1,electricity,kWh
+electricity_ev_res2,LD EV Residential L2,electricity,kWh
+electricity_ev_work1,LD EV Workplace L1,electricity,kWh
+electricity_ev_work2,LD EV Workplace L2,electricity,kWh
+electricity_ev_public2,LD EV Public L2,electricity,kWh
+electricity_ev_publicdcfc,LD EV Public DCFC,electricity,kWh
```
Comment (daniel-thom) [inline project/dimensions/enduses.csv:24]: nit: should these have `_dcfc`?

File: project/dimensions/subsectors.csv [hunk 2]
```diff
@@ -1,20 +1,56 @@
...
-phev_pickup,PHEV Pickup
-phev_suv,PHEV SUV
-bev_light_medium,BEV Light-Medium
-bev_medium,BEV Medium
-bev_non_freight_truck,BEV Non-freight truck
-bev_heavy,BEV Heavy
-bev_school_bus,BEV School Bus
-bev_transit_bus,BEV Transit Bus
-light_rail,Light Rail
+LDV+Personal_LDV+Compact+BEV_200, LDV+Compact+BEV_200
```
Comment (daniel-thom) [inline project/dimensions/subsectors.csv:3]: This may not be what @elainethale had in mind.

File: project/dimensions/enduses.csv (hunk 1 above)
Comment (ahcyip) [reply project/dimensions/enduses.csv:24]: sounds good. I will clean these up. (and add the mapping to match the actual data).

File: project/dimensions/scenarios.csv [hunk 3]
```diff
@@ -1,2 +1,4 @@
 id,name
-decarb_high,DECARB High
\ No newline at end of file
+decarb_high,DECARB High
+High+unmanaged_immediate,DECARB High and Unmanaged Immediate EV Charging
+High+unmanaged_smooth,DECARB High and Unmanaged Smooth EV Charging
```
[the remark this answers is not in the document]
Comment (ahcyip) [reply project/dimensions/scenarios.csv:4]: @elainethale and should I map unmanaged_smooth to blank?

[the remark this answers is not in the document]
Comment (elainethale) [reply project/dimensions/scenarios.csv:4]: Yes, please do that for now.

File: project/dimensions/subsectors.csv (hunk 2 above)
Comment (ahcyip) [reply project/dimensions/subsectors.csv:3]: I see - I'll revert this and create the dataset's dimensions and a mapping.

**Comment by ahcyip** [comment]
this is now outdated

**Pull request closed without merging**